### PR TITLE
fix: update dev setup docs and default env vars

### PR DESCRIPTION
**Pull request by JonBlauvelt**
## Description
After completing onboarding, making some changes to doc to reflect current setup process.  I made some assumptions here:
* We don't need/want any legacy (pre-azure) env vars
* We don't need to get keycloak creds directly since we're using the shared dev creds from the doc
## Related issues
https://github.com/Equal-Vote/star-server/issues/425

### CODE REVIEW

File: docs/contributions/1_local_setup.md [hunk 1]
```diff
@@ -97,7 +97,7 @@ Run these commands on your local machine:
    git clone --depth=1 https://github.com/YOUR_USER_NAME/star-server.git
    ```
 
-This will download the entire freeCodeCamp repository to your projects directory.
```
Comment (ArendPeter) [inline docs/contributions/1_local_setup.md:100]: Good catch!

File: docs/contributions/1_local_setup.md [hunk 2]
```diff
@@ -202,60 +186,26 @@ copy sample.env .env
...
 Launch backend 
 
 ```bash
 cd backend
+npm install
 npm run dev
 ```
 
-Launch frontend
+Launch frontend. If you want to just run the web client without setting up the server you can change the proxy under frontend/package.json to "proxy": "dev.star.vote".
```
Comment (ArendPeter) [inline docs/contributions/1_local_setup.md:201]: Were you able to get the proxy to work? 

I used to use this trick when we were on heroku, but I haven't had success with the dev.star.vote endpoint yet

**Review by ArendPeter — APPROVED** [review]
Looks good! Thanks for making these fixes

File: docs/contributions/1_local_setup.md (hunk 2 above)
Comment (JonBlauvelt) [reply docs/contributions/1_local_setup.md:201]: I didn't try it - assumed it was still working 😬 
I was able to get it to look at other local port here.

**Pull request merged**